src/import/ensureConfiguration.ts

```diff
@@ -283,6 +283,45 @@ function generateConfiguration(root: string, allowFromScratch: boolean): string
         .withSpaceBefore()
     );
   }
+  // index
+  // There are four possible locations for it (two filenames in two directories)
+  const inputIndexMarkdownPageContentPath = path.join(
+    root,
+    'ig-data',
+    'input',
+    'pagecontent',
+    'index.md'
+  );
+  const inputIndexXMLPageContentPath = path.join(
+    root,
+    'ig-data',
+    'input',
+    'pagecontent',
+    'index.xml'
+  );
+  const inputIndexMarkdownPagesPath = path.join(root, 'ig-data', 'input', 'pages', 'index.md');
+  const inputIndexXMLPagesPath = path.join(root, 'ig-data', 'input', 'pages', 'index.xml');
+  if (
+    !fs.existsSync(inputIndexMarkdownPageContentPath) &&
+    !fs.existsSync(inputIndexXMLPageContentPath) &&
+    !fs.existsSync(inputIndexMarkdownPagesPath) &&
+    !fs.existsSync(inputIndexXMLPagesPath)
+  ) {
+    // no index file found, create our default
+    setPairs.push(
+      new YAMLPair('indexPageContent', packageJSON.description ?? '')
+        .withCommentBefore(
+          getBoxComment(
+            'index.md',
+            'To use a provided index.md or index.xml file, delete the "indexPageContent" property below.'
```

Comment:
I think here it would be good to specify the location of the `index.md` file, aka `ig-data/input/pagecontent/index.[md | xml]`.

Reply:
I can add that!